README.rst

```diff
@@ -169,3 +169,11 @@ Contributing
 We happily welcome contributions to MLflow. We are also seeking contributions to items on the
 `MLflow Roadmap <https://github.com/mlflow/mlflow/milestone/3>`_. Please see our
 `contribution guide <CONTRIBUTING.md>`_ to learn more about contributing to MLflow.
+
+Committers
+----------
+
```

Comment:
```suggestion
<message>
```

Should we insert a message explaining who the listed users are?

Reply:
MLflow is currently maintained by Harutaka Kawamura <link>, Weichen Xu <link>, Corey Zumar <link> and Ben Wilson <link> with significant contributions from hundreds of exceptionally talented community members. 

something like that?

Reply:
made the change!